common/components/Footer/index.tsx

```diff
@@ -92,7 +93,21 @@ const LINKS_SOCIAL = [
 ];
 
 export default class Footer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { isOpen: false };
+  }
+
+  public toggleModal = () => {
+    this.setState({ isOpen: !this.state.isOpen });
+    document.body.classList.toggle('no-scroll');
```

Comment:
Not sure if I'm mistaking the intent here, but this should already be happening inside of the modal component: https://github.com/MyEtherWallet/MyEtherWallet/blob/develop/common/components/ui/Modal.tsx#L38

Reply:
Yup, `document.body.classList.toggle('no-scroll');` definitely is already happening inside the modal component and can be removed. The other half of `toggleModal()` (line 103) is using the state of the footer component to store a boolean to determine if the modal is open or closed, similar to how the [MnemonicDecrypt](https://github.com/MyEtherWallet/MyEtherWallet/blob/develop/common/components/WalletDecrypt/Mnemonic.tsx#L72) uses `seed` as a boolean to determine if the modal should be open or closed.